### PR TITLE
add conda stuff to docs

### PR DESCRIPTION
**Pull request by paulthebaker**
I checked the updated _Contributing_ section `conda` commands that install the dependencies by copy/pasting.

The main tests worked, but I got a bunch `black` errors from the linter on code that I didn't touch...

### CODE REVIEW

**Comment by paulthebaker** [comment]
The tests are passing, so there must be some difference in the configuration of the most recent version of black and what's being used in the tests.

**Review by AaronDJohnson — APPROVED** [review]
It might be nice to add a link to https://conda-forge.org/docs/user/tipsandtricks.html#installing-apple-intel-packages-on-apple-silicon for those that use M1 Macs in the installation instructions. This provides instructions on getting a conda environment using Rosetta to allow installation of both PINT and TEMPO2/libstempo. This looks great though. Thanks for updating this stuff, @paulthebaker !

**Comment by paulthebaker** [comment]
I'm currently trying to figure out issue #339 before merging this.

**Comment by aarchiba** [comment]
> The tests are passing, so there must be some difference in the configuration of the most recent version of black and what's being used in the tests.

The `black` stability policy basically says you need to bump your version of black once a year. https://black.readthedocs.io/en/stable/the_black_code_style/index.html#stability-policy

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -57,12 +57,41 @@ If you are proposing a feature:
...
+Start with an empty virtual environment, in this case called ``ent_dev``::
+
+    $ conda create -n ent_dev -y -c conda-forge python=3.9
+    $ conda activate ent_dev
+
+Now install things by running the commands::
+
+    $ conda install -c conda-forge enterprise-pulsar
+    $ conda remove enterprise-pulsar --force
+    $ conda install -c conda-forge  black=22.3.0 flake8 sphinx_rtd_theme pytest-cov
```
Comment (aarchiba) [inline CONTRIBUTING.rst:81]: Is there any reason not to put all the conda requirements on the `conda create` line? That should save some time.

**Comment by vhaasteren** [comment]
Hi @paulthebaker,

This is ready to merge. Any reason why not to quickly change the conda create line to the following?

```
$ conda create -n ent_dev -y -c conda-forge python=3.9 black=22.3.0 flake8 sphinx_rtd_theme pytest-cov
```

She is right that it's faster. Regardless, ready merge, so happy to pull the trigger

**Pull request merged**